### DISCUSSION TITLE
docker?

### DISCUSSION THREAD

**Discussion by ghost** [Elanis/web-to-desktop-framework-comparison#777] [Q&A] [answered]

**Comment by Elanis** [comment]
I'm not sure what you mean?

**Reply by Elanis** [reply]
That could be an idea, but I'm not sure if it would be usefull since Github Actions are already some kind of containers.
What would be the purpose of this test?

**Reply by Elanis** [reply, marked as answer]
Indeed, that could be useful for some people.
I will open an issue about this, but this won't be my priority 😄
